docs/markdown/Go/go.md

```diff
@@ -217,6 +217,16 @@ To add a timeout, set the `test_timeout` field to an integer value of seconds, l
 go_package(test_timeout=120)
 ```
 
+Similar as [with Python](doc:python-test-goal#timeouts), you can also set a default value and a maximum value in `pants.toml`:
```

Comment:
```suggestion
You can also set a default value and a maximum value in `pants.toml`:
```

Reply:
Probably good to remove this from other places too like Shell. Imo, no longer any need to mention Python